Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
Was this added manually, or did Cargo do it?

Also, I can't find any info on `v2` vs `v3`, can someone point me to the right links?

Reply:
@HCastano it was probably my fault but it is https://github.com/rust-lang/cargo/blob/master/CHANGELOG.md#changed

It's already on `nightly` so I think I triggered that ^^